modules/machineset-creating-efa-options.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-creating-efa-options_{context}"]
+= Creating machines that use an AWS Elastic Fiber Adapter
+
+You can deploy compute machines that use an AWS Elastic Fiber Adapter (EFA) by adding the `networkInterfaceType` field to the machine set YAML file for your compute machines.
+
+.Prerequisites
+
+* You have configured security groups to support the use of an EFA.
++
+Machines that use an EFA must belong to security groups that allow all traffic between all hosts in the security group. Clusters that use the Machine API are not configured to meet this requirement by default.
++
+You can manually configure your security groups to support the use of an EFA by using the AWS Management Console or the AWS CLI. For more information, see the Amazon EC2 documentation about https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#updating-security-group-rules[updating security group rules].
+
+.Procedure
+////
+//Per Huali and Zhaohua, only the steps to set `networkInterfaceType` are required. Therefore, I am not sure what to do with these steps:
+
+. Create a file named `unlimited-memlock.yaml` that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 02-worker-container-runtime <1>
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KZGVmYXVsdF91bGltaXRzID0gWwogICAgICAgICJtZW1sb2NrPS0xOi0xIgpdCg== <2>
+        mode: 420
+        overwrite: true
+        path: /etc/crio/crio.conf.d/10-custom <3>
+----
+<1> #Is this prescriptive or is it a user choice what to put here?#
+<2> #Not sure exactly what this specifies or what the user needs to fill it in, have a question out in Slack.#
```

Comment:
This is a base64 encoded string of the data for the file to be placed on disk. Might be worth including the decoded version as an example:
```
[crio.runtime]
default_ulimits = [
        "memlock=-1:-1"
]
```